scripts/validate-icons

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+ICONS_PATH='frontend/icons/src/main/resources'
+icon_files=$(git ls-files "$ICONS_PATH/*.svg")
+declared_icons=$(git grep -Pho '(?<=Icon) .* (?=\= loadIcon)' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java')
+
+for referenced_file in $(git grep -Pho '(?<=loadIcon\("/).*(?="\))' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java'); do
+  referenced_file_path="$ICONS_PATH/$referenced_file"
+  referenced_file_path_dark=${referenced_file_path::-4}_dark.svg
+  if [[ ! $icon_files =~ $referenced_file_path ]]; then
```

Comment:
`=~` is for regexes... can't it be a regular `!=` here?

Reply:
does `!=` work like contains?

Reply:
Oh!

Then some `*...*` could be useful

Reply:
whhat?

Reply:
```suggestion
  if [[ $icon_files != "*$referenced_file_path*" ]]; then
```